### DISCUSSION TITLE
how to Lure enemies to player?

### DISCUSSION THREAD

**Discussion by Misaka-Mikoto-G** [winstxnhdw/lc-hax#197] [Q&A] [answered]
M3+E not work

**Comment by winstxnhdw** [comment]
`M3` + `E` is for interacting with objects. `M3` alone can lure enemies already.

**Reply by Misaka-Mikoto-G** [reply]
i do not want control enemy,just let the enemy find the player,how can i do it?

**Reply by winstxnhdw** [reply, marked as answer]
Point at the player and click `M3`. It's in the README.
